iOSDevUK/Presentation/SecondaryViews/Sessions/SessionDetailView/SessionDetailView.swift

```diff
@@ -22,17 +22,10 @@ struct SessionDetailView: View {
     
     @ViewBuilder
     private func headerView() -> some View {
-        ZStack(alignment: .bottomLeading) {
```

Comment:
ok, will fix

Reply:
<img width="444" alt="Screenshot 2023-07-03 at 18 29 38" src="https://github.com/DavidFaraday/iOSDevUK/assets/5704018/ec954c75-06d0-4b86-8bb4-2e2594155fce">
I reverted changes for this screen, but I believe this is now hard to read, because of the white and pink on the blue background. WDYT?